src/sagas/reown.js

```diff
@@ -1112,7 +1120,13 @@ export function* onSessionDelete(action) {
  * @param {Object} action - The action containing the request payload
  */
 export function* onCreateNanoContractCreateTokenTxRequest(action) {
+  /* TODO: Restore this when we add back support for create nano contract create token tx
   yield* handleDAppRequest(action, ReownModalTypes.CREATE_NANO_CONTRACT_CREATE_TOKEN_TX);
+  */
+
+  // Reject all create nano contract create token tx requests for now
+  const { deny } = action.payload;
+  deny();
```

Comment:
We are currently not supporting this rpc method. We will add support in the future